tests/load/locustfile.py

```diff
@@ -89,7 +91,9 @@ def hello(self) -> None:
         with self._time_event(name="hello"):
             body = json.dumps(dict(messageType="hello", use_webpush=True))
             self.ws.send(body)
-            res = json.loads(self.ws.recv())
+            reply = self.ws.recv()
+            assert reply is not None, "No 'hello' response"
```

Comment:
It doesn't look like we'll get `None` out of `recv()`, only an empty str.
```suggestion
            assert reply, "No 'hello' response"
```